freud/density.pyx

```diff
@@ -24,6 +24,8 @@ cimport numpy as np
 # _always_ do that, or you will have segfaults
 np.import_array()
 
+ctypedef unsigned int uint
```

Comment:
Similarly, I would remove this typedef if it isn't needed to make the code work.

Reply:
Actually, it still fails in this place. It looks like they've got it working in some contexts, but not others (I would guess lvalues but not rvalues?). I'll remove it from the pxd but need to keep it here.